coe.c

```diff
@@ -1,8 +1,8 @@
+#include <unistd.h>
```

Comment:
- https://pubs.opengroup.org/onlinepubs/9699919799/functions/fcntl.html
- http://man.openbsd.org/fcntl
- https://www.freebsd.org/cgi/man.cgi?query=fcntl
- https://netbsd.gw.com/cgi-bin/man-cgi?fcntl++NetBSD-current

None of these state to #include <unistd.h>, but glibc man pages does. Cannot harm to add an extra header...